src/Microsoft.AspNetCore.Server.HttpSys/MessagePump.cs

```diff
@@ -32,7 +33,7 @@ internal class MessagePump : IServer
 
         private readonly ServerAddressesFeature _serverAddresses;
 
-        public MessagePump(IOptions<HttpSysOptions> options, ILoggerFactory loggerFactory)
```

Comment:
It's required for HttpSysServer, but only if auth is enabled. Same for IIS Integration. There are plenty of apps that don't need it though.